src/main/java/org/jenkinsci/plugins/docker/commons/credentials/DockerRegistryToken.java

```diff
@@ -48,6 +48,7 @@
     private final String email;
     private final String token;
 
+    private static final String charecterSet = "UTF-8";
```

Comment:
no space after the line.
The variable should be also upper-case according to the Oracle coding style